src/apps/api/views/sample.py

```diff
@@ -300,15 +300,22 @@ def get_samples_by_sequencing_centre(request):
     return finish_request(out)
 
 def get_updatable_fields_by_project(request, project):
-    project_lst = project.split(",")
-    project_lst = list(map(lambda x: x.strip(), project_lst))
-    # remove any empty elements in the list (e.g. where 2 or more comas have been typed in error
-    project_lst[:] = [x.lower() for x in project_lst if x] # Convert all strings in the list to lowercase
     out = list()
+    project_lst = d_utils.convertStringToList(project)
     
     for project in project_lst:
-        if project in lookup.DTOL_NO_COMPLIANCE_FIELDS:
-            out.append({project.upper(): lookup.DTOL_NO_COMPLIANCE_FIELDS[project]})
+        project = project.lower()
+
+        exported_fields = lookup.DTOL_EXPORT_TO_STS_FIELDS.get(project, [])
```

Comment:
it is for export, not for upload.  Can you get the fields from the manifest directly?  Another suggestion is to change the API to return compliance fields, not the updatable fields.